configure.ac

```diff
@@ -346,6 +346,16 @@ if test x$plugin_apimon = xyes; then
   AC_DEFINE_UNQUOTED(ENABLE_PLUGIN_APIMON, 1, "")
 fi
 
+AC_ARG_ENABLE([plugin_procdump],
+[AS_HELP_STRING([--disable-plugin-procdump],
+[Enable the windows process dumping plugin @<:@yes@:>@])],
+[plugin_procdump="$enableval"],
+[plugin_procdump="yes"])
+AM_CONDITIONAL([PLUGIN_PROCDUMP], [test x$plugin_procdump = xyes])
+if test x$plugin_procdump = xyes; then
+AC_DEFINE_UNQUOTED(ENABLE_PLUGIN_PROCDUMP, 1, "")
+fi
```

Comment:
Alignments in this block seems to be off

Reply:
Done